wrf.rb

```diff
@@ -28,6 +28,8 @@ def install
       (buildpath/"WPS").install resource("WPS")
   
       wrf_args = std_cmake_args + %w[
+        -DCMAKE_C_COMPILER=gcc
+        -DCMAKE_Fortran_COMPILER=gfortran
```

Comment:
I recommend completely removing the explicit dependency on GFortran. Homebrew's sanitized environment guarantees this will be true. There are currently no other fortran compilers installable via homebrew-core. Currently a user could install/link another compiler that CMake prioritizes higher than GFortran into `/usr/local/` and CMake will pick that up, but I consider this a bug in homebrew that is on my list to patch soon. (This happens because Homebrew is setting `CMAKE_PREFIX_PATH=/usr/local` when it should be enumerating (recursively) dependencies from `/usr/local/opt/<dependency>`.

```suggestion
```

Reply:
Yeah this is why I though of using this -- I have ifort installed (and this is usually my default compiler) but when building with Homebrew with the current formula CMake will pick `ifort` instead of `gfortran`. The fix I suggested solved the issue but it may be a bit of a hack...

Reply:
Where is ifort installed? It shouldn't be doing this... unless it's in `/usr/local`... at least on macOS. Are you on linux?